src/common/color_picker.c

```diff
@@ -148,446 +187,160 @@ static void color_picker_helper_4ch_seq(const dt_iop_buffer_dsc_t *const dsc, co
   const size_t off_mul = 4 * width;
   const size_t off_add = 4 * box[0];
 
-  const float w = 1.0f / (float)size;
+  _aligned_pixel macc = { { 0.0f, 0.0f, 0.0f, 0.0f } };
+  _aligned_pixel mmin = { { FLT_MAX, FLT_MAX, FLT_MAX, FLT_MAX } };
+  _aligned_pixel mmax = { { -FLT_MAX, -FLT_MAX, -FLT_MAX, -FLT_MAX } };
 
-  // code path for small region, especially for color picker point mode
-  if(cst_to == IOP_CS_LCH)
-  {
+#ifdef OPENMP_CUSTOM_REDUCTIONS
```

Comment:
As above, #if instead of #ifdef?

Reply:
These should be fixed in new commit. Thank you!